server/sonar-server/src/test/java/org/sonar/server/organization/ws/CreateActionTest.java

```diff
@@ -61,16 +65,19 @@
   @Rule
   public ExpectedException expectedException = ExpectedException.none();
 
+  private Settings settings = new MapSettings()
```

Comment:
Your suggestion implies that the unit test care about the default value mechanism. I don't think it does.

That's an implementation detail of `Settings`.

I don't see any benefit to rely on it, especially considering that changing the default value in the definition would break the UT even though it's totally unrelated. This would also make UT's code more obscure (here, it's explicite default value for all tests methods is `false`).

Property `ORGANIZATIONS_ANYONE_CAN_CREATE` is already overridden in tests when needed.